From hadi@cyberus.ca  Fri Feb  5 08:47:17 2010
From: jamal <hadi@cyberus.ca>
Date: Thu, 04 Feb 2010 14:50:56 -0500
Subject: NET: fix oops at bootime in sysctl code
To: Stefan Lippers-Hollmann <s.L-H@gmx.de>
Cc: stable@kernel.org, gregkh@suse.de, linux-kernel@vger.kernel.org, davem@davemloft.net
Message-ID: <1265313056.4651.18.camel@bigi>

This fixes the boot time oops on the 2.6.32-stable tree.  It is needed
only in this tree due to the divergance from upstream.

From: jamal <hadi@cyberus.ca>
Acked-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 kernel/sysctl_check.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/sysctl_check.c
+++ b/kernel/sysctl_check.c
@@ -220,6 +220,7 @@ static const struct trans_ctl_table tran
 	{ NET_IPV4_CONF_PROMOTE_SECONDARIES,	"promote_secondaries" },
 	{ NET_IPV4_CONF_ARP_ACCEPT,		"arp_accept" },
 	{ NET_IPV4_CONF_ARP_NOTIFY,		"arp_notify" },
+	{ NET_IPV4_CONF_SRC_VMARK,		"src_valid_mark" },
 	{}
 };
 
